import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1414 @@
+                start_pos = str(int(start_pos) + 1)
+
+        # resolve variant type, end position, and variant class
+        variant_type = resolve_vcf_variant_type(ref_allele, alt_allele)
+        end_pos = resolve_end_position(vcf_data, start_pos, variant_type, ref_allele)
+        variant_class = resolve_variant_classification(vcf_data, variant_type, ref_allele, alt_allele)
+
+    maf_data["Variant_Classification"] = variant_class
+    maf_data["Variant_Type"] = variant_type
+    maf_data["Reference_Allele"] = ref_allele
+    maf_data["Tumor_Seq_Allele1"] = ref_allele
+    maf_data["Tumor_Seq_Allele2"] = alt_allele
+    maf_data["Start_Position"] = start_pos
+    maf_data["End_Position"] = end_pos
+    return maf_data
+
+def create_maf_record_from_vcf(sample_id, center_name, sequence_source, vcf_data, is_germline_data, matched_normal_sample_id, tumor_sample_data_col):
+    """
+        Creates MAF record from VCF data.
+    """
+    # init maf record
+    maf_data = init_maf_record()
+
+    # set easy to resolve values
+    maf_data["Tumor_Sample_Barcode"] = sample_id
+    maf_data["Matched_Norm_Sample_Barcode"] = matched_normal_sample_id
+    maf_data["Center"] = center_name
+    maf_data["Hugo_Symbol"] = resolve_hugo_symbol(vcf_data)
+    maf_data["Chromosome"] = resolve_chromosome(vcf_data)
+    maf_data["Sequence_Source"] = resolve_sequence_source(vcf_data, sequence_source)
+    maf_data["Verification_Status"] = DEFAULT_VERIFICATION_STATUS
+    maf_data["FILTER"] = vcf_data.get("FILTER", "")
+    if is_germline_data:
+        maf_data["Mutation_Status"] = "GERMLINE"
+    else:
+        maf_data["Mutation_Status"] = DEFAULT_MUTATION_STATUS
+
+
+    # resolve counts and variant allele data
+    resolve_vcf_counts_data(vcf_data, maf_data, matched_normal_sample_id, tumor_sample_data_col)
+    resolve_vcf_variant_allele_data(vcf_data, maf_data)
+    resolve_vcf_matched_normal_allele_data(vcf_data, maf_data, matched_normal_sample_id)
+    return maf_data
+
+def is_valid_vcf_maf_record(maf_record):
+    return not ((maf_record["Reference_Allele"] == "N" or maf_record["Reference_Allele"] == "") and maf_record["Tumor_Seq_Allele2"] == "")
+
+def get_vcf_file_header(filename):
+    vcf_file_header = []
+    with open(filename, "r") as vcf_file:
+        for line in vcf_file.readlines():
+            if line.startswith("#CHROM"):
+                vcf_file_header = list(map(str.strip, line.replace("#", "").split("\t")))
+                break
+    if not vcf_file_header:
+        print("Could not extract VCF header from file: %s - Exiting..." % (filename))
+        sys.exit(2)
+    return vcf_file_header
+
+def extract_vcf_data_from_file(filename, center_name, sequence_source):
+    """ Load data from a VCF file. """
+    rejected_vars_by_varclass = 0
+    rejected_vars_by_mutstatus = 0
+    rejected_vars_by_inc_data = 0
+
+    (sample_id, tumor_sample_data_col, matched_normal_sample_id) = get_vcf_sample_and_normal_ids(filename)
```

Comment:
get_vcf_sample_and_normal_ids() is a call into the unreviewed block of code from pre-existing target processing tools (~308-971)
however, I did notice in that code that the column names "TUMOR" and "NORMAL" are hardcoded there as the placeholders for identifying the samples when the sample id is not in the file (presumably taken from the filename instead). This should also be part of the README ... that these keywords are required when the sample_id is not from the filename.